anax-in-container/Dockerfile.ubi.arm64

```diff
@@ -0,0 +1,47 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.0
+
+LABEL vendor="IBM"
+LABEL summary="The agent in a general purpose container."
+LABEL description="A container which holds the edge node agent, to be used in environments where there is no operating system package that can install the agent natively."
+
+ARG DOCKER_VER=19.03.8
+
+# yum is not installed, use microdnf instead
+RUN microdnf update -y --nodocs && microdnf clean all
+
+# add EPEL repo with jq pkg and all deps
+COPY EPEL.repo /etc/yum.repos.d
```

Comment:
This looks like a copy of the ppc docker file. Their dockerfile is slightly different to pick up jq. Is this what you intended?

Reply:
@dabooz Yes. The ppc version is already picking up jq from UBI repositories. Hence I used that Dockerfile as a template. I only adjusted the docker version to a newer one. I guess sometimes all Dockerfiles in that folder should be potentially consolidated. 😉